validate.py

```diff
@@ -151,6 +184,10 @@ def determine_if_mergeable():
   print('\nPASS')
 
 def determine_if_winner():
+  print('Points:')
+  for user, user_points in get_user_points().items():
```

Comment:
Actually, can we wrap this in a try/except so that we don't have to worry about wacky Exception win hacking :P

Reply:
Removed this section entirely.  For now this will just not print points.

#50 adds it back in a safe way